lapce-data/Cargo.toml

```diff
@@ -56,6 +56,7 @@ structdesc = { git = "https://github.com/lapce/structdesc" }
 lapce-core = { path = "../lapce-core" }
 lapce-rpc = { path = "../lapce-rpc" }
 lapce-proxy = { path = "../lapce-proxy" }
+bytemuck = "1.8.0"
```

Comment:
Could you please mark this as a platform-specific dependency?

Reply:
Eh, nevermind, piet-wgpu already uses bytemuck so it wouldn't do anything positive right now.